### PR TITLE
Update details blog

### PR DESCRIPTION
**Pull request by theshoplondon**

### CODE REVIEW

File: _posts/2024-08-27-updating-member-details.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+2. Is cheap and easy to get members to use
+3. Is intuitive, only taking members a few seconds to complete
+4. Can recalculate subscription rates based on the members’ current salary or circumstances
+5. Updates your membership database/CRM automatically
+
+Once you’ve got an improved ‘update your details’ service in place, you just need to send occasional emails or texts to members to remind them to keep their information updated (we recommend you do it about once a year). You can also include links on your website and other materials so members can update their record whenever convenient.
+
+### Try it for yourself
+
+We’ve put together a [demo](https://https://jtreview-pr-1167.herokuapp.com) so you can see how it works. Just add your email (your details won’t be stored) to get started, then try the update process for yourself.
```
Comment (rgarner) [inline _posts/2024-08-27-updating-member-details.markdown]: We can't currently say

> (your details won’t be stored)

As, y'know, they have to be, sort of.

File: _posts/2024-08-27-updating-member-details.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+**[Try the demo](https://https://jtreview-pr-1167.herokuapp.com)**
+
+### Why should unions do this?
+
+The Join Together approach to improving your ‘update your details’ process will lead to:
+
+- Better, more frequent contact and engagement with members
```
Comment (rgarner) [inline _posts/2024-08-27-updating-member-details.markdown]: I'd drop this a couple of places. Revenue and retention are our top two.

File: _posts/2024-08-27-updating-member-details.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+We’ve worked with unions for years, and heard all sorts of horror stories about the difficulties of staying in touch with members. To try and fix this, we’ve seen unions taking a variety of approaches. These include:
+
+- Asking for updated details by post or phone
+- Collecting information via forms or spreadsheets, usually just before a ballot takes place
+- Getting members to log in to a membership portal on the union website
+
+Each of these has different challenges and costs. Calls and letters are expensive to make or send, get low response rates and often interrupt what members are doing at a particular moment (”I’ll do that later” and “who’s that number calling me?”). 
+
+Collecting information via spreadsheets leads to enormous data protection risks, as well as problems with data entry, as files are passed from person to person, computer to computer, usually ending with a messy import or copy/paste/fix the mistakes process, slowing the whole thing down.
```
Comment (rgarner) [inline _posts/2024-08-27-updating-member-details.markdown]: Is this genuinely still a thing?

File: _posts/2024-08-27-updating-member-details.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Finally, union membership portals are challenging for their users. The typical experience involves using data members probably don’t know (their membership number or a password they never use, leading to a password reset or an email to the membership team) before logging into a piece of software they’re unfamiliar with to perform a task they can’t be sure they’ll be able to complete.
```
Comment (rgarner) [inline _posts/2024-08-27-updating-member-details.markdown]: This is sweet. They read well in this order (calls/letters->spreadsheets->portals). Is this the order that's most common -> least common?

**Review by rgarner — COMMENTED** [review]
Looks good, I think. It looks like it speaks directly to some problems that I can imagine existing, but I don't know if they do in 2024 so I'm taking your word for that

File: _posts/2024-08-27-updating-member-details.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  A better way to update union member details
+description:
+  Making it easy for members to update their details can result in better retention, more revenue and happier members. Here's how Join Together can help you do this.
+og_image_path: 
+date:   2024-08-27 11:19:13 +0000
+---
+
+We’ve worked with unions for years, and heard all sorts of horror stories about the difficulties of staying in touch with members. To try and fix this, we’ve seen unions taking a variety of approaches. These include:
```
Comment (tekin) [inline _posts/2024-08-27-updating-member-details.markdown]: Are there really "Horror stories" about "...staying in touch with members"? If there are, should we say what they are? Or are the horror stories actually about trying to keep member data up to date?

**Pull request merged**